client/globals.d.ts

```diff
@@ -22,3 +22,7 @@ declare const wcpaySettings: {
 	};
 	currentUserEmail: string;
 };
+
+interface Window {
```

Comment:
Let's drop it and use `wcTracks` instead of `window.wcTracks` where it's required.